src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -566,6 +566,22 @@ namespace Trinity
     };
 
     template<class Check>
+    struct PlayerListSearcherByPosition
```

Comment:
No, it's not. 

        PlayerListSearcherByPosition(uint32 phaseMask, std::list<Player*> &objects, Check & check)
            : i_phaseMask(phaseMask), i_objects(objects), i_check(check) { }

vs

        PlayerLastSearcher(WorldObject const* searcher, Player*& result, Check& check) : i_phaseMask(searcher->GetPhaseMask()), i_object(result), i_check(check)
        {
        }

Primarily the difference is all other searchers need an object and derive phasemask (and usually position) from the object. This has phasemask passed as argument.

Unless, I'm misunderstanding the objection.